recipe.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Healthy Protein Pancakes</title>
+</head>
+<body>
+    <h2><Healthy Protein Pancake/h2>
+    <img
+    src= "https://www.erinliveswhole.com/wp-content/uploads/2021/04/proteinpancakes-5.jpg"
```

Comment:
This is missing a specified attribute, which is why it's failing. Refer to the test specs.

Reply:
solved.  thanks

Reply:
also added the alt="Protein Pancakes!"